src/sources/hamonisedAttributes.json

```diff
@@ -0,0 +1,39 @@
+{
+    "physicalSpecimenId": {
+        "displayName": "Physical Specimen ID"
+    },
+    "ods:physicalSpecimenIdType": {
+        "displayName": "Physical Specimen ID Type"
+    },
+    "ods:organizationId": {
+        "displayName": "Organization",
+        "annotate": {
```

Comment:
What does the `annotate` do?

Reply:
The idea was to include configuration properties in this object for when annotating this harmonised property.
For example: the organizationId field will be bound to the organizations participating in DiSSCo, via the JSON or some other source in the future, I can determine that the form field type when annotating should be a select dropdown containing all the organizations as options. This would also probably only be applicable when using 'adding' or 'correcting' as a motivation.

Reply:
Ok yes good to think ahead :+1: